mobile/Makefile

```
@@ -0,0 +1,26 @@
+.PHONY: xcodebuild run install boot-sim generate clean
+
+DEVICE = ${DEVICE_ID}
+ifndef DEVICE_ID
+	DEVICE=$(shell xcrun simctl list devices 'iOS' | grep -v 'unavailable' | grep -v '^--' | grep -v '==' | head -n 1 | cut -d ' ' -f 7 | sed 's/[()]//g')
```

Aha! This line was snatching "3rd" out of this line:

```
    iPhone SE (3rd generation) (F647334F-D7C1-4BD6-9B5F-0E3D9713C02B) (Shutdown)
```

Here's one way to more deterministically get the ID (at least, it works for me 😄):

```suggestion
	DEVICE=$(shell xcrun simctl list devices 'iOS' | grep -v 'unavailable' | grep -v '^--' | grep -v '==' | head -n 1 | python3 -c "import re, sys; m = re.search(r'\((\w+?\-\w+?\-\w+?\-\w+?\-\w+?)\)', sys.stdin.readline().strip()); print(m.group(1))")
```

🤔 Now it runs and displays console output...but I don't get a visible simulator window showing up anywhere.

---

Yeah, the gray screen is the same I ended up with. The system that spawns the menu runs, but the button doesn't seem to render.

Regarding the whole setup: i ve been trying to move from `cargo-apk` and a makefile (+ a bunch of ios specific files) to `xbuild`. It's working for android, but not for iOS yet. Hopefuly that will get rid of all these extra files.

---

Yeah, the gray screen is the same I ended up with. The system that spawns the menu runs, but the button doesn't seem to render.

Regarding the whole setup: i ve been trying to move from `cargo-apk` and a makefile (+ a bunch of ios specific files) to `xbuild`. It's working for android, but not for iOS yet. Hopefuly that will get rid of all these extra files.

---

I think `xbuild` with it's cross platform `x devices` would be much nicer than requiring python3 and the makefile.

---

Instead of using python here, would it not also work to match more precisely on the UUID using sed or grep (something like `grep -E -o -i "([0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12})"`)?

---

The `grep` line from above works as expected for your device `    iPhone SE (3rd generation) (F647334F-D7C1-4BD6-9B5F-0E3D9713C02B) (Shutdown)`